index.html

```diff
@@ -54,9 +54,13 @@
         <h1>NSLondon</h1>
         <p>A group for iOS and OS X developers who want to explore all thats
         new and fascinating in cocoa development. Expect deep dives and
-        bleeding edge stuff. We meet in London, but if you don't happen to be
+        bleeding edge stuff.</p>
+        
+        <p>We meet in London, but if you don't happen to be
         there, <a href="http://vimeopro.com/user20904333/nslondon/" target="_blank">catch the videos.</a></p>
 
+        <p>If you are interested in speaking at an event in the future, then please <a href="https://docs.google.com/forms/d/e/1FAIpQLSfkp5A9WagbojPpE114eDKi4svQCRdbjPh857MX9w6dxBaTIg/viewform?c=0&w=1" target="_blank">submit this form.</a></p>
```

Comment:
To agree with the comment @doug made on Slack, the link text should be more descriptive (for accessibility purposes) so that screen readers make sense.